tests/whitelist.txt

```diff
@@ -1,9 +1,5 @@
 integration.client.test_runner
 integration.client.test_standard
-integration.fileserver.test_fileclient
-integration.fileserver.test_roots
-integration.loader.test_ext_grains
-integration.loader.test_ext_modules
```

Comment:
It looks like these last two lines shouldn't be removed. These tests are still here:

https://github.com/saltstack/salt/blob/develop/tests/integration/loader/test_ext_grains.py
https://github.com/saltstack/salt/blob/develop/tests/integration/loader/test_ext_modules.py